exotica_core/src/problems/dynamic_time_indexed_shooting_problem.cpp

```diff
@@ -81,6 +81,10 @@ void DynamicTimeIndexedShootingProblem::Instantiate(const DynamicTimeIndexedShoo
 
     ApplyStartState(false);
     ReinitializeVariables();
+
+    // set start and goal states
+    set_X_star(init.GoalState.replicate(1, T_));
```

Comment:
Do we want the entire problem to have the goal state or do we only want the state to be at the final timestep and then propagated back in the value function? That's different problems!

Reply:
See above.